lib/src/components/data-table/drag-and-drop/DragAndDropContainer.tsx

```diff
@@ -0,0 +1,49 @@
+import { useDataTable } from '../DataTableContext'
+import {
+  closestCenter,
+  DndContext,
+  DragOverlay,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors
+} from '@dnd-kit/core'
+import { restrictToVerticalAxis } from '@dnd-kit/modifiers'
+import { arrayMove, sortableKeyboardCoordinates } from '@dnd-kit/sortable'
+
+import * as React from 'react'
+
+export const DragAndDropContainer: React.FC = ({ children }) => {
+  const { idColumn, onDragAndDrop, order, setData } = useDataTable()
+
+  const sensors = useSensors(
+    useSensor(PointerSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates
+    })
+  )
+
+  function handleDragEnd(event) {
```

Comment:
Super minor, just for consistency

```suggestion
  const handleDragEnd = (event) => {
```